src/Symfony/Bundle/WebProfilerBundle/Controller/ExceptionController.php

```diff
@@ -58,24 +58,8 @@ public function showAction(string $token)
         $this->profiler->disable();
 
         $exception = $this->profiler->loadProfile($token)->getCollector('exception')->getException();
-        $template = $this->getTemplate();
 
-        if (!$this->twig->getLoader()->exists($template)) {
-            return new Response($this->errorRenderer->getBody($exception), 200, ['Content-Type' => 'text/html']);
-        }
-
-        $code = $exception->getStatusCode();
-
-        return new Response($this->twig->render(
```

Comment:
Marked the controller as internal in #32695 so the twig dependency can easily be removed in master